Sources/Basic/TerminalController.swift

```diff
@@ -78,11 +78,13 @@ public final class TerminalController {
             return width
         }
 
+#if TIOCGWINSZ
```

Comment:
This isn't going to work as expected... nothing is going to set this build condition (Swift isn't importing C preprocessor macros as build conditions).

What platform is this a problem on?